docs/source/notebook.md

```diff
@@ -4,19 +4,25 @@
 
 ## Introduction
 
-The notebook extends the console-based approach to interactive computing in
-a qualitatively new direction, providing a web-based application suitable for
-capturing the whole computation process: developing, documenting, and
-executing code, as well as communicating the results. The Jupyter notebook
-combines two components:
-
-**A web application**: a browser-based tool for interactive authoring of
-documents which combine explanatory text, mathematics, computations and their
-rich media output.
-
-**Notebook documents**: a representation of all content visible in the web
-application, including inputs and outputs of the computations, explanatory
-text, mathematics, images, and rich media representations of objects.
+**Jupyter Notebook** is a notebook authoring application, under the [Project
```

Comment:
"**Jupyter Notebook** is a notebook authoring application" is a circular definition, it refers back to the term being defined (notebook). 

Would something else non-circular like "web-based notebook environment for interactive computing" or "interactive computing environment that enables users to author notebook documents" or "web application for creating and sharing computational documents" work better here?

Reply:
Jupyter (the umbrella project) has a lot of problems with naming ambiguity, which I don't think we can solve here..."Jupyter Notebook" is the name of the application, which creates "notebook documents", and notebook documents are defined and described just below this section to add clarity. I think the extra words here just increase complexity and make the text harder to read.

Reply:
That makes sense